configure

```diff
@@ -150,22 +149,22 @@ updateVault = ->
     process.stderr.write stderr
     rest()
 
-git.long (str) ->
-  if str != ""
-    version = str.substring 0, 8
-  else
-    version = try (fs.readFileSync path.join(__dirname, './VERSION'), 'utf-8').trim()
-
-  if argv.hostname
-    options.hostname = argv.hostname
-    updateVault()
-  else if os.type() is "Darwin"
-    options.hostname = "dev.koding.com"
+try
+  version = require('git-rev-sync').short()
```

Comment:
`version = str.substring 0, 8` is required - there are other places where we apply string expansions